code/game/machinery/washing_machine.dm

```diff
@@ -231,50 +231,25 @@
 		istype(W,/obj/item/clothing/shoes) || \
 		istype(W,/obj/item/clothing/suit) || \
 		istype(W,/obj/item/bedsheet))
-
-		//YES, it's hardcoded... saves a var/can_be_washed for every single clothing item.
-		if( istype(W,/obj/item/clothing/suit/space ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/suit/syndicatefake ) )
-			to_chat(user, "This item does not fit.")
-			return
-//		if( istype(W,/obj/item/clothing/suit/powered ) )
-//			to_chat(user, "This item does not fit.")
-//			return
-		if( istype(W,/obj/item/clothing/suit/cyborg_suit ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/suit/bomb_suit ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/suit/armor ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/suit/armor ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/mask/gas ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/mask/cigarette ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/head/syndicatefake ) )
-			to_chat(user, "This item does not fit.")
-			return
-//		if( istype(W,/obj/item/clothing/head/powered ) )
-//			to_chat(user, "This item does not fit.")
-//			return
-		if( istype(W,/obj/item/clothing/head/helmet ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/gloves/furgloves ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if(istype(W, /obj/item/clothing/gloves/color/black/krav_maga/sec))
-			to_chat(user, "<span class='warning'>Washing these gloves would fry the electronics!</span>")
-			return
+		
+		var/unwashable = list(
+			/obj/item/clothing/suit/space,
+			/obj/item/clothing/suit/syndicatefake,
+			/obj/item/clothing/suit/cyborg_suit,
+			/obj/item/clothing/suit/bomb_suit,
+			/obj/item/clothing/suit/armor,
+			/obj/item/clothing/mask/gas,
+			/obj/item/clothing/mask/cigarette,
+			/obj/item/clothing/head/syndicatefake,
+			/obj/item/clothing/head/helmet,
+			/obj/item/clothing/gloves/furgloves,
+			/obj/item/clothing/gloves/color/black/krav_maga/sec,
+			/obj/item/clothing/shoes/magboots
+		)
+		for(var/X in unwashable)
+			if(istype(W,X))
+				to_chat(user, "<span class='warning'>This item cannot be washed.</span>")
+				return
```

Comment:
there is a helper for this, `is_type_in_list`

Reply:
Cool